### ISSUE TITLE
Make Table of Contents one view

### ISSUE THREAD

**Issue by socvid** [mimbrown/render#122]
Faisal wishes that the table of contents would be more like ones found in books.  He showed how books list all of the different parts in order of appearance.  Our app first lists the major parts (Intro, Matthew, Mark, etc), and then a new window opens with a complete table of contents for that part.  What if this was an accordion list view?  Meaning that instead of the table of contents for each part to open in a new window, they would open below the selected book?  Another idea Faisal had was for the new window for each part to open as a popup.

I personally like the accordion list view idea.  At the same time, this strikes me as being a pretty subjective view, and I would be curious to hear other local's feedback on this.

**Comment by kuzunguka** [comment]
I agree that 1. the accordion design idea for the ToC is a pretty subjective item and 2. I like it!  Let's see if we can get more feedback from local reviewers.
